pom.xml

```diff
@@ -756,11 +682,10 @@
 
                                         <echo message="Update ignite.version in C++" />
                                         <replaceregexp byline="true" encoding="UTF-8">
-                                            <regexp pattern="(AC_INIT.+\[)\d+\.\d+\.\d+.*?(\].+)" />
+                                            <regexp pattern="(project\(.*\s)\d+\.\d+\.\d+.*?(\))" />
```

Comment:
Did you try it out? I'm not sure it will replace correctly.

Reply:
Yes, I try it, as I mentioned in comment in jira. You can try it by yourself or in on-line checker tool.

Reply:
`mvn -P update-versions -Dnew.ignite.version=<new_version> validate` and check that in CMakeLists.txt everything correctly updates.